app/teachingCall/teachingCallStatus/controllers/ModalAddInstructorsCtrl.js

```diff
@@ -189,6 +193,32 @@ teachingCallApp.controller('ModalAddInstructorsCtrl', this.ModalAddInstructorsCt
 		return true;
 	};
 
+	$scope.areAllLecturersInvited = function() {
+		if (!$scope.startTeachingCallConfig.invitedInstructors) {
+			return true;
+		}
+
+		for (var i = 0; i < $scope.startTeachingCallConfig.invitedInstructors.length; i++) {
```

Comment:
(Fat arrows are in all our browsers except IE. If we end up with a customer using IE, we could investigate transpiling.)